scripts/schemagen.go

```diff
@@ -83,17 +85,21 @@ func main() {
 		log.Fatal(err)
 	}
 
-	err = exec.Command("gofmt", "-w", fileName).Run()
+	fmtd, err := format.Source(buf.Bytes())
 	if err != nil {
 		log.Fatal(err)
```

Comment:
I can't pull up specific examples right now, but I'm pretty sure the generator may sometimes produce invalid code that will fail to format and therefore fail to be even written in a file. I think it's more desirable to actually let the generator write invalid code into the file and eventually fix it manually.

TL;DR I would turn the `log.Fatal()` here into `log.Print()` or something like that.